### PR TITLE
Spencer

### PR DESCRIPTION
**Pull request by spencekarl**
Created my page. Please review. Thanks.

---

This change is [Review on Reviewable](https://reviewable.io/reviews/nw0428/sibspring2016/3)

### CODE REVIEW

File: spencer.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <link rel="stylesheet" type="text/css" href="css/spencer/style.css">
+  <link href="http://fonts.googleapis.com/css?family=Oswald:400" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+  <article>
+    <h1>SPENCER LOVES ADVENTURE</h1>
+  </article>
+
+  <video autoplay loop id="video-background" muted>
```
Comment (nw0428) [inline spencer.html:13]: This doesn't seem to work on IE 7!
Then again who cares about IE 7? =)

**Comment by nw0428** [comment]
I like the use of HTML5. I didn't know about the video tag.
LGTM

**Comment by melissa0831** [comment]
lgtm

**Comment by JQuezada0** [comment]
lgtm

**Comment by jdorancy** [comment]
LGTM

**Comment by rhymiz** [comment]
Looks good to me!

**Pull request merged**